homeassistant/components/netgear_lte/services.py

```diff
@@ -0,0 +1,86 @@
+"""Services for the Netgear LTE integration."""
+from typing import TYPE_CHECKING
+
+import voluptuous as vol
+
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant, ServiceCall
+from homeassistant.helpers import config_validation as cv
+
+from .const import (
+    ATTR_AUTOCONNECT,
+    ATTR_FAILOVER,
+    ATTR_HOST,
+    ATTR_SMS_ID,
+    AUTOCONNECT_MODES,
+    DOMAIN,
+    FAILOVER_MODES,
+    LOGGER,
+)
+
+if TYPE_CHECKING:
+    from . import LTEData, ModemData
```

Comment:
These two should be in there own file to avoid circular references.
Maybe `models.py`?

Reply:
I would not right now. This is going to be replaced with a coordinator so I do not see the point in moving it when I plan on replacing.

Reply:
OK, not compulsory right now.